src/doc.ts

```diff
@@ -0,0 +1,300 @@
+import { Events } from './events';
+import { Scene } from './scene';
+import { DownloadWriter, FileStreamWriter } from './serialize/writer';
+import { ZipWriter } from './serialize/zip-writer';
+import { Splat } from './splat';
+import { serializePly } from './splat-serialize';
+import { localize } from './ui/localization';
+
+// ts compiler and vscode find this type, but eslint does not
+type FilePickerAcceptType = unknown;
+
+const SuperFileType: FilePickerAcceptType[] = [{
+    description: 'SuperSplat document',
+    accept: {
+        'application/octet-stream': ['.super']
```

Comment:
![image](https://github.com/user-attachments/assets/22281aab-036e-4395-918e-43ee859234f9)

Reply:
Given there is already .ksplat and .splat, I think users will confuse .ssplat with a runtime format.

Reply:
Actually I think that's the worst possible option.

Reply:
Bribery didn't work, we're going with `.ssproj` extension.